rrfs-test/testinput/rrfs_fv3jedi_letkf_2022052619.yaml

```diff
@@ -32,7 +32,7 @@ background:
         filename_sfcd: 20220526.190000.sfc_data.nc
         filename_sfcw: 20220526.190000.fv_srf_wnd.res.nc
         filename_cplr: 20220526.190000.coupler.res
-      pattern: %mem%
+      pattern: "%mem%"
```

Comment:
Curious, what is the differences using \" or not?

Reply:
The quotes are needed because a value cannot start with a `%` if it is to be read by the yaml tools. See #186.

Reply:
`%` is a special character in YAML. Here the `pattern` key expects a `string` type of value. Usually, YAML will not require a string value to be quoted (This is actually a huge drawback of YAML causing many problems and is criticized by many experts).  If a value starts with special characters or has special meaning needing to be escaped, we need to put on the quotes explicitly. Another example, if we want to take a string value of "True" instead of a boolean value, we need to add quotes as well.